prometheus/observer.go

```diff
@@ -0,0 +1,50 @@
+// Copyright 2016 The Prometheus Authors
```

Comment:
Nit, but afaik the year of the Apache license header should reflect the year a file as been created. Same in other files below.